.github/workflows/roles-readme.yml

```
@@ -0,0 +1,55 @@
+---
+name: create roles readme
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - master
```

Let's only do this when the argument-spec changes. I think there's no need to do it on every push to master.

```suggestion
      - master
      paths:
      - ''roles/**/meta/argument_specs.yml''

```